src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/talismans/MagicianTalisman.java

```diff
@@ -40,12 +42,20 @@ public MagicianTalisman(SlimefunItemStack item, ItemStack[] recipe) {
         addItemSetting(allowEnchantmentBooks);
```

Comment:
You forgot to add it to the item
```suggestion
        addItemSetting(allowEnchantmentBooks, allowCustomEnchantments);
```